src/main/java/uk/gov/ons/validation/service/ProcessWranglerQuestionData.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.ons.validation.service;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.lambda.AWSLambdaAsync;
+import com.amazonaws.services.lambda.AWSLambdaAsyncClient;
+import com.amazonaws.services.lambda.model.InvokeRequest;
+import com.amazonaws.services.lambda.model.InvokeResult;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.gov.ons.validation.common.Constants;
+import uk.gov.ons.validation.entity.QuestionInputData;
+import uk.gov.ons.validation.entity.ValidationConfig;
+import lombok.extern.log4j.Log4j2;
+import uk.gov.ons.validation.entity.WranglerRequest;
+import uk.gov.ons.validation.entity.WranglerResponseData;
+import uk.gov.ons.validation.util.PropertiesUtil;
+
+import java.util.List;
+
+import static java.lang.String.format;
+
+@Log4j2
+public class ProcessWranglerQuestionData {
```

Comment:
This class would ideally have a set of unit tests

Reply:
I discussed with @dllmorgan yesterday and decided to extend unit test case next time.

Reply:
The problem is that "next time" never happens. Next time, you would have worked on something new and would have forgotten your thought process when you originally worked on this code. It seems the only testing at the moment is uploading it to AWS and checking that it still runs as expected. The point of unit tests is to assert that the code does what it should as early as possible. Unit tests should not be an extra, they should be as important if not more important than the code itself.